integration/storagetest/maptests.go

```diff
@@ -193,3 +196,64 @@ func (*mapTests) TestTileRoundTrip(ctx context.Context, t *testing.T, ms storage
 		t.Fatalf("ReadWriteTransaction(): %v", err)
 	}
 }
+
+func (*mapTests) TestGetSetRoundTrip(ctx context.Context, t *testing.T, ms storage.MapStorage, as storage.AdminStorage) {
```

Comment:
```suggestion
func (*mapTests) TestSetGetRoundTrip(ctx context.Context, t *testing.T, ms storage.MapStorage, as storage.AdminStorage) {
```

You need to set before you can get :-)

Reply:
:-)